### DISCUSSION TITLE
在”新窗口打开PDF"时，无法使用标注中添加的颜色

### DISCUSSION THREAD

**Discussion by WeiGuangRang** [MuiseDestiny/zotero-style#511] [General]
我发现一个小问题，就是我之间打开zotero时，自带的pdf阅读器可以使用自己添加的标注颜色，但是使用单独的pdf阅读窗口时，就只能使用默认的标注颜色，我想了解一下这个算不算一个问题，还是就是这样的，谢谢大家

**Comment by MuiseDestiny** [comment]
正常的，毕竟修改颜色功能不是官方支持的，而是插件通过特别手段支持的。
